packages/labs/task/src/test/task_test.ts

```diff
@@ -107,6 +107,9 @@ suite('Task', () => {
     return new A();
   };
 
+  // TODO: the name of this function is confusing. It doesn't wait for the task
+  // to complete. It just waits a rAF. That's why it can be used to check that
+  // a task is pending.
   const tasksUpdateComplete = nextFrame;
```

Comment:
Good point. I'd consider dropping this in favor of just directly using `nextFrame`